frontend/src/components/Pages/InspectionPage/InspectionTable.tsx

```diff
@@ -195,7 +228,19 @@ const InspectionRow = ({ inspection, openDialog, setMissions, openScheduledDialo
     )
 }
 
-const columns = ['Status', 'Name', 'Description', 'Area', 'Last completed', 'Deadline', 'Add to queue']
+const SmallScreenInfoText = () => {
+    const { TranslateText } = useLanguageContext()
+    return (
+        <div id="SmallScreenInfoText">
```

Comment:
Consider maybe create a StyledSmallScreenInfoText with the styles for normal and when the screen is small instead of the div. The div is fine tho, only if you want to minimize the use of divs